Composer/packages/client/src/components/NavLinks/index.tsx

```diff
@@ -5,9 +5,9 @@
 
 import { jsx } from '@emotion/core';
 import React, { Fragment } from 'react';
+import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
 
-import { dialogItem } from '../../pages/language-understanding/styles';
-
+import { dialogItemSelected, dialogItemNotSelected } from '../../pages/language-understanding/styles';
```

Comment:
This seems like a smell to me. Should these styles be defined next to nav links?